### PR TITLE
Satellites doc updates

### PR DESCRIPTION
**Pull request by brandonSc**
Some things in the satellites doc page were out of date.

### CODE REVIEW

File: docs/cloud/satellites.md [hunk 1]
```diff
@@ -67,11 +67,14 @@ Follow the steps in the [Earthly Cloud overview](./overview.md#getting-started)
 
 ### 2. Purchase a Satellites Plan
 
-Satellites are now a paid feature and require a valid subscription to begin using. The subscription includes a 14-day free trial which can be canceled before any payment is made. Visit the [pricing page](https://earthly.dev/pricing) for more billing details.
+Satellites are a paid feature which require a subscription to begin using. The subscription includes a 14-day free trial which can be canceled before any payment is made. Visit the [pricing page](https://earthly.dev/pricing) for more billing details.
+
+You can start your free trial by using the checkout form below. Be sure to provide the name of your Earthly org from Step 1.
```
Comment (vladaionescu) [inline docs/cloud/satellites.md:72]: I see that this now includes a new field for the org. I should probably update the automated email in Zapier so that it no longer asks for this.

File: docs/cloud/satellites.md [hunk 2]
```diff
@@ -156,12 +159,7 @@ For more information on managing satellites, see the [Managing Satellites page](
 
 ## Satellite specs
 
-Satellites are currently only available in one size, and it has the following specs:
-
-* 4 CPUs
-* 16 GB of RAM
-* 90 GB of cache storage
-* 5 Gib internet bandwidth
+Satellites are currently only available in different sizes and architectures. Please see the [Pricing Page](https://earthly.dev/pricing) for more details.
```
Comment (vladaionescu) [inline docs/cloud/satellites.md:162]:
```suggestion
Satellites are available in various sizes and architectures. Please see the [Pricing Page](https://earthly.dev/pricing) for more details.
```

File: docs/guides/auth.md [hunk 3]
```diff
@@ -118,6 +118,19 @@ git:
...
+You can run earthly with `--verbose`, which will provide debugging messages to help understand how a remote earthly reference is transformed into a git URL for cloning.
+
+You can additionally enable low-level git debugging in buildkit, by adding the following to your `~/.earthly/config.yml`:
+
+```yaml
+global:
+  buildkit_additional_args: [ '-e', 'BUILDKIT_DEBUG_GIT=1' ]
+```
+
+The buildkit logs can be displayed with `docker logs earthly-buildkitd`.
```
Comment (vladaionescu) [inline docs/guides/auth.md:132]: This change seems like it belongs to another PR? Is this intentional?

File: docs/cloud/satellites.md (hunk 1 above)
Comment (brandonSc) [reply docs/cloud/satellites.md:72]: Yes, I was thinking that too.

File: docs/guides/auth.md (hunk 3 above)
Comment (brandonSc) [reply docs/guides/auth.md:132]: I think I scooped it from main by accident. But I don't see why this wouldn't be in the 0.7 docs. @alexcb is this okay with you?

Comment (brandonSc) [reply docs/guides/auth.md:132]: I also tried bringing 0.7-docs up to date with main. I think I was used to the `next` branch. Apologies if I shouldn't have done that

Comment (vladaionescu) [reply docs/guides/auth.md:132]: > I also tried bringing 0.7-docs up to date with main. I think I was used to the next branch. Apologies if I shouldn't have done that

We only do that on release. `main` might contain docs for unreleased features.

**Review by vladaionescu — APPROVED** [review]

File: docs/guides/auth.md (hunk 3 above)
Comment (brandonSc) [reply docs/guides/auth.md:132]: That makes sense. Sorry, I did that a bit impulsively. It looks like this is the only doc change from `main` that wasn't in `docs-0.7` yet, and doesn't correspond to an unreleased feature. 

Is the process for adding to these docs documented anywhere yet? Perhaps it can go in `CONTRIBUTING.md`. Happy to add that.

Comment (vladaionescu) [reply docs/guides/auth.md:132]: Already there: https://github.com/earthly/earthly/blob/main/CONTRIBUTING.md#documentation ;-)

Comment (brandonSc) [reply docs/guides/auth.md:132]: Heh I skimmed, but missed it. Should've used ctrl-f :)

**Comment by brandonSc** [comment]
I snuck in a couple more satellite doc updates:
* Using the `EARTHLY_SATELLITE` environment var (as per user request)
* Encourage docker login to avoid rate limits

File: docs/guides/auth.md (hunk 3 above)
Comment (alexcb) [reply docs/guides/auth.md:132]: > I think I scooped it from main by accident. But I don't see why this wouldn't be in the 0.7 docs. @alexcb is this okay with you?

Seems like you just need to change this PR to be against `docs-0.7`; currently you're PR is attempting to merge `docs-0.7` (plus additional commits) into `main` -- if this is merged as-is (please dont!), github will squash commits together, which will then lead to a conflict when we merge `main` back into `docs-0.7`.

**Review by alexcb — APPROVED** [review]
